### PR TITLE
Add unit tests for HFSS and EDB.

### PR DESCRIPTION
**Pull request by MaxJPRey**
Add unit test for spiral lumped port with non-collinear objects.
Add test for edb with dxf.
It is impossible to create a wave port without faces. An exception is raised prior the creation.

### CODE REVIEW

File: pyaedt/hfss.py [hunk 1]
```diff
@@ -3085,7 +3085,7 @@ def create_wave_port_from_sheet(
                 faces = sheet
             else:
                 faces = self.modeler.get_object_faces(sheet)[0]
-            if not faces:
+            if not faces:  # pragma: no cover
                 self.logger.error("Wrong Input object. it has to be a face id or a sheet.")
                 return False
```
Comment (akaszynski) [inline pyaedt/hfss.py:3090]: This should be a `TypeError`.

**Review by akaszynski — APPROVED** [review]
Approved with suggestions.

File: pyaedt/hfss.py (hunk 1 above)
Comment (MaxJPRey) [reply pyaedt/hfss.py:3090]: Agreed. It will be part of a global refactoring work.

**Review by PipKat — APPROVED** [review]
@MaxJPRey 
Hope I didn't hold you up.

**Comment by MaxJPRey** [comment]
> @MaxJPRey Hope I didn't hold you up.

Not at all @PipKat . It even allowed @akaszynski and I to talk about a refactoring.

**Pull request merged**